homeassistant/components/switcher_kis/__init__.py

```diff
@@ -65,6 +67,16 @@
     }
 )
 
+SERVICE_TURN_ON_WITH_TIMER_NAME = "turn_on_with_timer"
+SERVICE_TURN_ON_WITH_TIMER_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_ENTITY_ID): cv.entity_id,
```

Comment:
We only use `CONF_` constants in config schemas not in service schemas. Please use `ATTR_` constants instead.
```suggestion
        vol.Required(ATTR_ENTITY_ID): cv.entity_id,
```